koku/api/report/ocp/ocp_query_handler.py

```diff
@@ -41,19 +42,28 @@ def __init__(self, query_parameters, url_data,
             kwargs    (Dict): A dictionary for internal query alteration based on path
         """
         provider = 'OCP'
-        kwargs['provider'] = provider
+        self._initialize_kwargs(kwargs)
```

Comment:
The `_initialize_kwargs()` function is also a workaround for initialization ordering problems.